### ISSUE TITLE
Feature request: Minimap similar to VS2015 / Sublime Text

### ISSUE THREAD

**Issue by BlackStar-EoP** [edbee/edbee-lib#62]
Instead of the regular scrollbar, I would like to use a minimap similar to the one in Sublime Text / Visual Studio.

**Comment by vadi2** [comment]
I can't say I've ever had a practical use for these things myself. It's a cool idea but it just doesn't add anything.

**Comment by sebcaux** [comment]
It could be nice. But for me it is a low priority add-on.

**Comment by BlackStar-EoP** [comment]
I agree with the both of you that it doesn't add that much functionality, however, from work experience I often find it handy to scroll through a file looking for a certain pattern in the minimap. For me it's low prio as well :)
